libs/langchain-mongodb/langchain_mongodb/chat_message_histories.py

```diff
@@ -112,7 +114,12 @@ def __init__(
             self.client = client
         elif connection_string:
             try:
-                self.client = MongoClient(connection_string)
+                self.client = MongoClient(
+                    connection_string,
+                    driver=DriverInfo(
+                        name="Langchain", version=version("langchain-mongodb")
```

Comment:
for version lets add an identifier for chat_message_history

Reply:
@prakul What do you have in mind? chat_message_history is a module in langchain-mongodb..